core/src/zisk_rom_2_asm.rs

```diff
@@ -5970,8 +5992,13 @@ impl ZiskRom2Asm {
                     new_pc,
                     ctx.comment_str("pc = c(const) + jmp_offset1")
                 );
-                ctx.jump_to_static_pc =
-                    format!("\tjmp pc_{:x} {}\n", new_pc, ctx.comment_str("jump to static pc"));
+                // Check if target address exists in ROM before generating static jump
+                if rom.sorted_pc_list.binary_search(&new_pc).is_ok() {
+                    ctx.jump_to_static_pc =
+                        format!("\tjmp pc_{:x} {}\n", new_pc, ctx.comment_str("jump to static pc"));
+                } else {
+                    ctx.jump_to_dynamic_pc = true;
+                }
```

Comment:
Adding this because when we process the program headers as riscv instructions, they may produce valid riscv instructions. deadbeef for example is a JAL instruction to a label that is out of bounds. This is here so that that label gets resolved using dynamic jump.